src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -221,3 +221,13 @@ The span finishes once the operation has been executed. The span `status` is set
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and `SentryEvent` will be linked when viewing it on the **Issue Details** page in [sentry.io](https://sentry.io).
 
 Learn more about our [file I/O integration](/platforms/dart/configuration/integrations/file/).
+
+### sqflite Database Instrumentation
+
+The sqflite Database Instrumentation starts a span out of the active transaction, bound to the scope for each `CRUD` operation.
```

Comment:
```suggestion
The sqflite database instrumentation starts a span from an active transaction that's bound to the scope of each `CRUD` operation.
```

Reply:
Replaced the `out of the active transaction...` to from an active transaction...` everywhere.